### DISCUSSION TITLE
Herd Pro (windows) - customizing Mysql configuration ?

### DISCUSSION THREAD

**Discussion by GL-MorganC** [beyondcode/herd-community#563] [Q&A] [answered]
Hey,

Is there any solution, when we add mysql services on Herd Pro (windows) to modify the my.ini file? 

Didn't see the file and how to proceed to do this... if anyone has an idea?

Thank's for your help;

**Comment by johnluxor** [comment, marked as answer]
You can create `my.ini` file in mysql directory 

`C:\Users\[USERNAME]\.config\herd\bin\services\mysql\8.0.36 and set necessary options`

**Comment by GL-MorganC** [comment]
Perfect! thank's for your help !
